### ISSUE TITLE
Mockup of Each Webpage of the new Nashville FCC Website

### ISSUE THREAD

**Issue by bestyoucanbe** [nashvillefcc/nashvillefcc2020website#1]
GIVEN that a new Nashville FCC Website is to be designed
WHEN the software developer needs to design a webpage
THEN the software developer will be provided with a Mockup of the webpage (that has been generated by a designer and approved by the Project Manager beforehand)

**Comment by bestyoucanbe** [comment]
![FCCInitialDesign](https://user-images.githubusercontent.com/48490636/76250428-5947cd80-6213-11ea-8ae7-31889d740931.jpg)

**Comment by bestyoucanbe** [comment]
The above is the initial hand-drawn design of the website.  Each complete box is a page that shows up when you visit the site.  The first box (top-down order) is a splash screen that "fades out" and you land on the second box.  The @ symbol with a tail is a transition to the next page.  The bottom right corner box is a view of the third box as seen on a smaller device (ex:  phone).

**Comment by bestyoucanbe** [comment]
As project managers for this project, these are the ideas that we have kicked around at our last major discussion on this project.  Use this as a guide to determine if we/you need to enter new issues to complete the MVP (Minimum Viable Product) or enter new Stretch goals--new "stretch goal" label is now available when entering a issue.  Stretch Goals will only be attempted after the MVP is completed.  Any issues entered will need to be approved by a project manager/assistant project manager before it becomes a To Do item for the team.  If you are not sure if an issue has been approved by a Project Manager, please ask before attempting to code it (otherwise it may not be approved at Pull Request time/Merge to Master time).  My goal--Brainstorm ideas by using multiple brains continuously to make the site better!
-------------------------------------------------------------------------------------------------------

Nashville FCC Website

MVP Content 
 • Mockup of Website
  -Do we have someone who is willing to do this now or are we still recruiting? What’s his/her name and contact information? [Ellen offered if we can’t find someone else.]
How long are we waiting on a designer? If need be we could come up with a bare bones frame for the site and start there.
 • link together socials
  -Does this mean set-up links for social events or set-up links to social websites (Facebook, Twitter, LinkedIn, Meetup, Slack, etc) ?
 • sponsors 
-Do we have anyone except the Nashville Tech Council?  Anybody else interested so far?
	- TechFed (monies)
	- Nashville Tech Council (room)
	- C2A === Sponsor Us (C2A--Call to action)
		- Swag
		- Food
		- Software Yearly Subscription (Giveways)
		- etc
How much of the page will be used for sponsors?
? Sep page or one page design
Purpose/Audience: Sponsors, freeCodeCampers, new devs (training)

 • about us and how we started
-Laura--You seem to know the history of this place...can you create a blurb about this section for us?
Seth - fCCNashville overall
Laura - MentorNight
 • monthly vs weekly description
-Do we want to have these sections on the same page?  Do we anticipate changing the periodicity of these meetings in the future (ex: will we experiment with having a large quarterly gathering in the future or bring in special “celebrity speakers” for a special event)?  Is it better to have an Upcoming Events Page instead that unifies all the events (but can denote the weekly/monthly/quarterly/special in the title of the event in some special way (ex:  Font/Color)?
 ◦ integration with meetup to display next event for each MVP Design 
-See questions on previous section
• Saltstack.com website (design inspiration)

What is the plan with this?  How do we intend to use this?

Stretch 
• presentations and link to speakers
-Is this going to be done for every event?
-Is it true to assume that the link to speakers and presentations are after the event has happened?  Do we intend to have a “trailer” to advertise/promote upcoming events/speakers/presentations?
 -Stretch stretch goal---Facebook Live (TwitchTV)


• Additional stretch goal???-->Make the website responsive
-Can we see the content on a smartphone/tablet?

• Additional stretch goal???-->Make a link to the speakers presentations?


Basic Build MVP 
• HTML, CSS, JS 
• ReactJS and Netlify

-Build it in React and see how it works.  Have never used Netlify...may/want help with this when we get it that far.



Open Source 

• Mentor Night Led Project
🌝<--Love it!

Saving data
How do we want to save data. JSON file? Firebase?
ERD
Do we want to build our own backend? What language would we build it in?



MVP
Meetup.com
Links to social: Discord, Slack, Twitter, Facebook Group
Responsiveness
Content
Sponsors
Events
About
Open source

**Comment by Hellrungj** [comment]
Here are the Wireframes for Mobile and Desktop:

![Overview Mobile](https://user-images.githubusercontent.com/8736564/80660845-7dc16880-8a52-11ea-885d-26fe15643a67.PNG)
![Overview Desktop](https://user-images.githubusercontent.com/8736564/80660846-7e59ff00-8a52-11ea-9b12-99740b3c84b0.PNG)

**Comment by bestyoucanbe** [comment]
Thanks @Hellrungj !  👍  This will help guide us!  @nashvillefcc/nashvillefcc2020webdevs  -- FYI

**Comment by MoribundMedium** [comment]
Marking this issue as closed. Mockups were created, they look good, and give us a place to start. We'll revisit them when the current ones have exceeded their use.
